build-support/gen_build_version.py

```diff
@@ -23,7 +23,9 @@
 def get_version():
     version = os.getenv("STARROCKS_VERSION")
     if not version:
-        version = "UNKNOWN"
+        git_res = subprocess.Popen(["git", "symbolic-ref", "-q", "--short" ,"HEAD"], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
we already have equivalent ability in `build.sh`, https://github.com/StarRocks/starrocks/blob/main/build.sh#L46

not necessary to do it again here.